platform/branding-registry-impl/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -15,17 +15,9 @@
 <blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
            xmlns:cm="http://aries.apache.org/blueprint/xmlns/blueprint-cm/v1.1.0">
 
-    <bean id="brandingRegistry" class="org.codice.ddf.branding.impl.BrandingRegistryImpl">
-        <property name="brandingPlugins" ref="brandingPlugins"/>
+    <bean id="brandingRegistry" class="org.codice.ddf.branding.impl.BrandingRegistryImpl" init-method="init" destroy-method="destroy">
     </bean>
 
     <service ref="brandingRegistry" interface="org.codice.ddf.branding.BrandingRegistry"/>
 
-    <bean id="brandingPlugins" class="org.codice.ddf.platform.util.SortedServiceList">
```

Comment:
❓ Why wasn't the SortedServiceList working? Is it an issue with the SortedServiceList itself that needs to be fixed?

Reply:
I'm not sure. It seems to be working in other places, but was not working here.

Reply:
Hm... ok. Would be nice if we didn't have to duplicate that logic everywhere 😕